services/static-webserver/client/source/class/osparc/ui/markdown/Markdown.js

```diff
@@ -123,10 +123,10 @@ qx.Class.define("osparc.ui.markdown.Markdown", {
       }
       if (domElement && domElement.children) {
         const elemHeight = this.__getChildrenElementHeight(domElement.children);
-        if (this.getMaxHeight() && elemHeight > this.getMaxHeight()) {
-          this.setHeight(elemHeight);
+        if (this.getMaxHeight() && (elemHeight > this.getMaxHeight())) {
+          this.setHeight(elemHeight + 40);
```

Comment:
Where is this 40 coming from?

Reply:
It seems to be padding or there is some strange behavior coming in from the markdown plugin as there isn't any code adding it from our side